src/IotCentral/IotCentral/IotCentral/NewAzureRmIotCentralApp.cs

```diff
@@ -60,7 +60,7 @@ public class NewAzureRmIotCentralApp : IotCentralBaseCmdlet
             HelpMessage = "Pricing tier for IoT Central applications. Default value is S1.")]
         [PSArgumentCompleter("S1")]
         [ValidateNotNullOrEmpty]
-        public string Sku { get; set; }
+        public PSIotCentralAppSku Sku { get; set; }
```

Comment:
This is a breaking change, and ignores the argument completer - please add the new known values to the ArgumentCompleter, and implement the defaulting logic accordingly

Reply:
I've implemented default logic, and as argument completer would loop through the values of PSIotCentralAppSku right now, I just remove the PSArgumentCompleter

Reply:
@markcowl , after discussion with our team, we think it's ok to have this "breaking change", when the customer install this new version of powershell, they would be notify to input correct value to create an IoTCentral app